WineDependencies.md

```diff
@@ -21,15 +21,15 @@ Add the repository:
 
 |For this version: | Use this command:          
 |------------------|--------------------------------
-|Debian 9 Jessie   | sudo add-apt-repository 'deb https://dl.winehq.org/wine-builds/debian/ jessie main'
-|Debian 10 Buster  | sudo add-apt-repository 'deb https://dl.winehq.org/wine-builds/debian/ buster main'
-|Debian 11 Bullseye| sudo add-apt-repository 'deb https://dl.winehq.org/wine-builds/debian/ bullseye main'
 |Ubuntu 20.10      | sudo add-apt-repository 'deb https://dl.winehq.org/wine-builds/ubuntu/ groovy main'
 |Ubuntu 20.04      | sudo add-apt-repository 'deb https://dl.winehq.org/wine-builds/ubuntu/ focal main'
 |Ubuntu 19.10      | sudo apt-add-repository 'deb https://dl.winehq.org/wine-builds/ubuntu/ eoan main'
 |Ubuntu 19.04      | sudo apt-add-repository 'deb https://dl.winehq.org/wine-builds/ubuntu/ disco main'
 |Ubuntu 18.10      | sudo apt-add-repository 'deb https://dl.winehq.org/wine-builds/ubuntu/ cosmic main'
 |Ubuntu 18.04<br>Linux Mint 19.x | sudo apt-add-repository 'deb https://dl.winehq.org/wine-builds/ubuntu/ bionic main'
+|Debian 9 Jessie   | sudo add-apt-repository 'deb https://dl.winehq.org/wine-builds/debian/ jessie main'
+|Debian 10 Buster  | sudo add-apt-repository 'deb https://dl.winehq.org/wine-builds/debian/ buster main'
```

Comment:
Correct me if I'm wrong, but I don't think add-apt-repository exists on Debian out of the box.